### DISCUSSION TITLE
TypeError: setup() got an unexpected keyword argument 'numerical_features'

### DISCUSSION THREAD

**Discussion by UngaBungaCoder** [pycaret/pycaret#2159] [Q&A] [answered]
pycaret version 2.3.6
python version 3.8.12

can anyone tell me how to fix this ? 
![ภาพ](https://user-images.githubusercontent.com/99654968/153864264-1a12dc07-f6a5-4e96-966b-5dbcba51c082.png)

Thanks !

**Comment by ngupta23** [comment, marked as answer]
First place to check would be the docstring (usually visible when you hover over the function name). You can also do the following to get the docstring:

```python
?setup
```

Based on that, it should be `numeric_features` not `numerical_features`
